tests/tlslite-ng.json

```diff
@@ -40,6 +40,9 @@
          {"name" : "test-ecdhe-rsa-key-exchange.py"},
          {"name" : "test-ecdhe-rsa-key-exchange-with-bad-messages.py"},
          {"name" : "test-empty-extensions.py"},
+         {"name" : "test-encrypt-then-mac-renegotiation-crash.py",
+          "comment" : "Test requires renegotiation support",
+          "exp_pass" : false},
```

Comment:
even if `Encrypt-then-MAC renegotiation crash` is expected to fail, the `sanity` should not

see for other scripts that `-e`xclude the problematic part